src/components/RelatedVideoItem/RelatedVideoItem.test.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import RelatedVideoItem from './RelatedVideoItem.component';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}));
+
+const mockedVideo = {
+  id: { kind: 'youtube#video', videoId: 'nmXMgqjQzls' },
+  snippet: {
+    title: 'Video Tour | Welcome to Wizeline Guadalajara',
+    description:
+      'Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...',
+    thumbnails: {
+      default: {
+        url: 'https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg',
+        width: 120,
+        height: 90,
+      },
+    },
+  },
+};
+
+const mockedVideos = [
+  {
+    id: { kind: 'youtube#video', videoId: 'nmXMgqjQzls' },
+    snippet: {
+      title: 'Video Tour | Welcome to Wizeline Guadalajara',
+      description:
+        'Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...',
+      thumbnails: {
+        default: {
+          url: 'https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg',
+          width: 120,
+          height: 90,
+        },
+      },
+    },
+  },
+];
```

Comment:
It seems the only element in the array is the same as mockedVideo var value, you could use it like:
```suggestion
const mockedVideos = [ mockedVideo ];
```
And also it would be better to move mocked data into a separate file to keep the test as clean as possible.